Rachnera Message Bust.rb

```diff
@@ -114,7 +114,7 @@ def draw(x, y, face_name, face_index, deadzone_x = nil, deadzone_h = nil, fade_r
       @bust_overflow.bitmap = nil
       if deadzone_x && deadzone_h
         extra_width = bust_bitmap.width - @bust.width
-        extra_height = Graphics.height - deadzone_h - @bust.y
+        extra_height = Graphics.height - (overflow_behind_text ? @bust.y : @bust.y + deadzone_h)
```

Comment:
The fact `deadzone_h` is effectively unused for everyone's but Balia suggests we're due for some refactoring. But no need to do it right now.

Reply:
Yeah, Balia is a special case because she has semitransparent pixels in the gradient area, which means there's a visible seam when those are doubled up below it by the overflow sprite. I didn't find anyone else with that problem, and her wispy bits aren't important, so I figured it's an acceptable workaround.